main/exercise/exercise_submit.php

```diff
@@ -1261,7 +1275,7 @@ function save_now(question_id, url_extra, validate) {
 
             // Only for the first time
             var dataparam = "'.$params.'&type=simple&question_id="+question_id;
-            dataparam += "&"+my_choice+"&"+hotspot+"&"+remind_list+"&"+my_choiceDc;
+            dataparam += "&"+my_choice+"&"+hotspot+"&"+remind_list+"&"+my_choiceDc+"&"+my_choice_agree+"&"+my_choice_disagree+"&"+my_choice_agree_scale+"&"+my_choice_agree_reorder;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Line exceeds maximum limit of 120 characters; contains 180 characters](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)